Source/WebCore/html/HTMLLinkElement.cpp

```diff
@@ -615,17 +615,28 @@ std::optional<LinkIconType> HTMLLinkElement::iconType() const
     return m_relAttribute.iconType;
 }
 
+// https://html.spec.whatwg.org/multipage/links.html#linkTypes
+static bool isLinkedToExternalResource(LinkRelAttribute relAttribute)
```

Comment:
Naming suggestion: `linksToExternalResource(attr)`

Reply:
Based on @rniwa's comment, It seems we only need to collect URLs for subresources that may be fetched based on https://html.spec.whatwg.org/multipage/semantics.html#fetching-and-processing-a-resource-from-a-link-element, so I updated this to `mayFetchResource`. Let me know if you think there is better naming for this.